docs/index.md

```diff
@@ -2,16 +2,27 @@
 
 CodeSee helps development teams visually understand how your large scale codebase works, document it and collaborate continuously.
 
-Before proceeding with setup, you can sign up for a CodeSee account [here](https://app.codesee.io/register).
+**CodeSee is early beta software**, so thank you so much for trying it out, and please let us know (support@codesee.io) what you think, or how we can help!
 
 ## Overview
-Today, CodeSee consists of a few pieces. As part of your install, you will:
+By integrating CodeSee with your app, you will be able to make and view recordings of the innerworkings of your Javascript or Typescript application.
 
-* add two npm libraries to your javascript app:
-    * a CodeSee babel plugin `@codesee/babel-plugin-instrument`, which instruments your code to send data to the CodeSee server
-    * the CodeSee Tracker library `@codesee/tracker`, which actually sends the data to the CodeSee server
+In the instructions below, we will:
+
+1. Install two npm packages, and
+2. Configure babel so that CodeSee instruments your code when running in development
+
+Once installed, whenever you run your app in development and view it in your browser, you should see a circle with the CodeSee logo.
+
+- If it's red, click it to log into CodeSee
+- If it's purple, you're ready to go! Click it to start recording, interact with your app, then click it again to stop recording.
+
+A few thoughts to help you on your CodeSee journey:
+
+- Keep recordings as short as possible -- focus on the one thing you want to understand better.
+- When you stop recording, CodeSee will try to open a new tab with your recording in it shortly. For larger recordings or on slower networks, there can be some buffering, so please give it a few seconds or more to finish sending the recording to our servers.
+- The CodeSee UI can be a little sluggish with larger recordings! Sorry about that! For the moment, we ask that you give our UI a second to process now and then. But fear not, we take performance very seriously and we're actively working on performance improvements.
```

Comment:
should we add a note about popup blockers?

Reply:
Fixed!